src/main/scala/io/getclump/Sources.scala

```diff
@@ -10,70 +11,70 @@ protected[getclump] trait Sources extends Tuples {
    * the key used to get that value.
    */
   def source[KS, V, K](fetch: KS => Future[Iterable[V]])(keyExtractor: V => K)
-                      (implicit cbf: CanBuildFrom[Nothing, K, KS]): ClumpSource[K, V] =
+                      (implicit cbf: CanBuildFrom[Nothing, K, KS], ec: ExecutionContext): ClumpSource[K, V] =
```

Comment:
I see now, it's because of the future compositions for method adaption. Ideally, we should be able to avoid the EC here, but I think it isn't too bad to keep like this.